google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/MessageDispatcher.java

```diff
@@ -78,12 +80,12 @@
   private final LinkedBlockingQueue<String> pendingNacks = new LinkedBlockingQueue<>();
   private final LinkedBlockingQueue<String> pendingReceipts = new LinkedBlockingQueue<>();
 
-  private final Lock alarmsLock;
-  // The deadline should be set by the subscriber connection before use,
-  // but set it to some reasonable value just in case.
-  private final AtomicInteger messageDeadlineSeconds = new AtomicInteger(10);
-  private ScheduledFuture<?> ackDeadlineExtensionAlarm;
-  private ScheduledFuture<?> pendingAcksAlarm;
+  // The deadline should be set before use. Here, set it to something unreasonable,
+  // so we fail loudly if we mess up.
+  private final AtomicInteger messageDeadlineSeconds = new AtomicInteger(-1234);
```

Comment:
I think it would be better to set it to a reasonable default. I think it would be better to be defensive rather than fail loudly in the client library.

Reply:
Done